R/migrations.R

```diff
@@ -0,0 +1,102 @@
+run_migration <- function(queue, output_dir) {
```

Comment:
As discussed:

* don't write any files (I think this means you can omit `output_dir` as an argument)
* dry run mode so it's easy to test on production (i.e., don't write to redis)
* drop the first two parts to migrate_task